ProductApp/src/main/java/com/stack/jspdemo/repository/ProductRepositoryCustom.java

```diff
@@ -0,0 +1,12 @@
+package com.stack.jspdemo.repository;
+
+import java.util.List;
+
+import com.stack.jspdemo.model.Product;
+
+public interface ProductRepositoryCustom {
+	List<Product> getAllProducts();
+
+	void insertProduct(Product product);
```

Comment:
why not using `save()` method? this one is already provided by `CrudRepository` and `JpaRepostory`